README.md

```diff
@@ -1,4 +1,44 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go`
+
+## get package and deps
+
+```
+go get github.com/golemfactory/bootstrap_go
+cd ~/go/src/github.com/golemfactory/bootstrap_go/
+# get dependencies and test dependencies
+go get -v -t ./...
+```
+
+## run
+```
+go run main/main.go 
+```
+
+## tests
+
+```
+go test -v -race ./...
+```
+where `-race` enables builtin race detector 
```

Comment:
I'd recommend simple `go test ./...` here.
`-v` adds verbosity which isn't often necessary
`-race` adds additional overhead to each run, which slows down the development
so the default with any options is good enough

Reply:
ok